app/Mage.php

```diff
@@ -178,64 +180,65 @@ public static function getVersionInfo()
      * @link https://openmage.github.io/supported-versions.html
      * @link https://semver.org/
      */
-    public static function getOpenMageVersion(): string
+    public static function getOpenMageVersion(bool $withHas = false): string
     {
-        $info = self::getOpenMageVersionInfo();
-        $versionString = "{$info['major']}.{$info['minor']}.{$info['patch']}";
-
-        if ($info['stability'] && $info['number']) {
-            return "{$versionString}-{$info['stability']}.{$info['number']}";
-        }
-        if ($info['stability']) {
-            return "{$versionString}-{$info['stability']}";
+        $versionfromGit = self::getVersionFromGit($withHas);
+        if ($versionfromGit) {
+            return $versionfromGit;
```

Comment:
Wouldn't that collide with other repos that uses openMage?

if my repo has tags, then it would return my Repo tags instead of the OpenMage tags

Reply:
Missed to push last commits. Check for composer version has to come first. 

Will update PR tomorrow.

Reply:
We have to ways to install ...

- git clone ... gibt branch + hash is shown
- composer ... branch is shown
- having a fork with own tags ... your tag will be shown (you know what you have forked)